.github/workflows/ci-unit-tests.yml

```diff
@@ -43,9 +43,12 @@ jobs:
 
       - name: "Build"
         run: |
-          poetry install --no-root
+          poetry install --no-root -E colbert
           poetry build
 
       - name: Run ragstack-ai unit tests
+        env:
+          COLBERT_ASTRA_TOKEN: ${{ secrets.COLBERT_ASTRA_TOKEN }}
+          COLBERT_ASTRA_SCB: ${{ secrets.COLBERT_ASTRA_SCB }}
```

Comment:
We cannot rely on an astra database like this in the CI 

we should use cassio, and run the integration tests after the db creation in _run_e2e_tests.yaml file

additionally, to make it easier, we can spin up a docker container with dse. You can copy and paste this class 
https://github.com/datastax/ragstack-ai/blob/main/ragstack-e2e-tests/e2e_tests/test_utils/cassandra_container.py

Reply:
The integration test uses a Cassandra container now.